packages/cosmic-swingset/lib/ag-solo/start.js

```diff
@@ -100,33 +101,15 @@ async function buildSwingset(
   });
   config.vats.set('timer', { sourcepath: getTimerWrapperSourcePath() });
 
-  // 'storage' will be modified in-place as the kernel runs
-  const storage = buildStorageInMemory();
-  config.hostStorage = storage.storage;
-
-  // kernelStateFile is created in init-basedir.js, should never be missing
-  const lines = new readlines(kernelStateFile);
-  let line;
-  while ((line = lines.next())) {
-    const [key, value] = JSON.parse(line);
-    config.hostStorage.set(key, value);
-  }
+  const { storage, commit } = makeSimpleSwingStore(basedir, kernelStateDBName, false);
```

Comment:
Should we put the last parameter in an option bag, like `{ clobber: false }` or `{ useExisting: true }` or something? I'm not sure it's necesary, but it's not obvious at this call site what that flag might be doing, and giving it a name might help.

Reply:
I feel like the API is so simple as is that complicating with an options bag would just be complicating it.